guides/doc-Administering_Project/master.adoc

```diff
@@ -4,7 +4,7 @@ include::common/header.adoc[]
 
 = {AdministeringDocTitle}
 
-ifdef::satellite[]
+ifeval::["{build}" == "satellite" && "{backend}" == "html5"]
```

Comment:
This is a lot of duplication. Can this `ifeval` be placed in a file to include? Would that be an assembly?

Reply:
You mean to place it inside the feedback module file?

Reply:
I'd prefer to keep it this simple way, it's easiest to read in the source code...

Reply:
I say no.

Reply:
@maximiliankolb thoughts on this?

Reply:
There's a third option: place this into your file:

```
ifeval::["{build}" == "satellite" && "{backend}" == "html5"]
...
my content
...
endif::[]
```

similar to the firewalld snippet.

Reply:
I think it's best if the condition is visible in master.adoc for orientation in the code.